pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -162,20 +162,7 @@ func GetAdvisedCloneStrategy(sc *storagev1.StorageClass) (cdiv1.CDICloneStrategy
 	return strategy, found
 }
 
-func isLocalStorageOperator(sc *storagev1.StorageClass) bool {
-	_, found := sc.Labels["local.storage.openshift.io/owner-name"]
-	return found
-}
-
-func knownNoProvisioner(sc *storagev1.StorageClass) bool {
-	return isLocalStorageOperator(sc)
-}
```

Comment:
Why get rid of this?

Reply:
I think we should do it for all SCs with `provisioner == kubernetes.io/no-provisioner` and not only with those labeld with `local.storage.openshift.io/owner-name`.

Reply:
I think it was pretty opinionated in https://github.com/kubevirt/containerized-data-importer/pull/2226, thought I can't remember the exact reason
Let me dig through the discussions to find out

Reply:
https://github.com/kubevirt/containerized-data-importer/pull/2226#issuecomment-1101290228
I am particularly worried about this change going live for whatever setups are out there in the wilderness.
So unless you insist I would drop it and have a separate PR just discussing this allegedly non trivial change

Reply:
sure, no problem - re-adding.